{"name": "Login with valid credentials", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 200\n +  where 403 = <Response [403]>.status_code", "trace": "api_client = <helpers.api_client.APIClient object at 0x000001EC82D89E50>\n\n    @pytest.fixture\n    def registered_user(api_client):\n        user_data = {\n            'email': generate_email(),\n            'password': generate_password(),\n            'name': generate_name()\n        }\n        # Регистрация\n        response = api_client.post(Urls.REGISTER, json=user_data)\n>       assert response.status_code == 200\nE       assert 403 == 200\nE        +  where 403 = <Response [403]>.status_code\n\nconftest.py:22: AssertionError"}, "start": 1755429716212, "stop": 1755429716212, "uuid": "438aa9ea-ab33-449d-a2d1-406c7147fce2", "historyId": "635e39da8edc9086fbdae58401159c07", "testCaseId": "635e39da8edc9086fbdae58401159c07", "fullName": "tests.test_login_user.TestLoginUser#test_login_valid_user", "labels": [{"name": "feature", "value": "User Login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "dea-1611"}, {"name": "thread", "value": "7772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}